### DISCUSSION TITLE
NODE_F401RE not found. Please ensure the device is correctly connected.

### DISCUSSION THREAD

**Discussion by Laiaborrell** [TheAlbertDev/MASB#41] [Q&A] [answered]
I have the board plugged in and operating because it emits light. I get the following error (which I attach below) every time I upload the program. 

![placa](https://user-images.githubusercontent.com/78802600/107700326-9d5f1000-6cb7-11eb-8c9e-e7b674b5b651.jpeg)

![error](https://user-images.githubusercontent.com/78802600/107700089-48bb9500-6cb7-11eb-87f3-4f110fb6a1e7.PNG)

I have searched for the error on the internet, but I have not been able to solve it.

**Comment by Albert-Alvarez** [comment, marked as answer]
When you connect the board, does a window open as if you had connected a pen drive? When you try to upload the program, does the LED COM blink?

It is difficult to give you solutions without more information 😅 I give you some tests that you can do before we look at it in the next session if you can't figure it out:

- Try using another USB port.
- Use another USB cable (there are cables that are only for charging and do not include the data bus).
- Make sure you pick the appropriate upload method: Upload method > Mass Storage.
- Reinstall/update the board drivers in the Arduino IDE.
- Reinstall Arduino IDE.

**Comment by Laiaborrell** [comment]
No, it does not appear as a pen drive was connected. Maybe the USB cable is just for charging. I've tried with another cable and it works.
Thankyou

**Comment by Albert-Alvarez** [comment]
> No, it does not appear as a pen drive was connected. Maybe the USB cable is just for charging. I've tried with another cable and it works.
> 
> Thankyou

Glad to hear that the proposed solutions have helped you to solve the problem 😉
